### PR TITLE
Implement none attestation authenticator

### PR DESCRIPTION
**Pull request by majk-p**
Resolves https://github.com/webauthn4j/webauthn4j/issues/851

### CODE REVIEW

**Comment by majk-p** [comment]
@ynojima please have a look :pray:

File: webauthn4j-test/src/main/java/com/webauthn4j/test/authenticator/webauthn/NoneAttestationAuthenticator.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+// as described in https://www.w3.org/TR/webauthn-2/#sctn-none-attestation
+public class NoneAttestationAuthenticator extends WebAuthnModelAuthenticator {
+
+    public NoneAttestationAuthenticator(
+            AAGUID aaguid,
+            KeyPair attestationKeyPair,
+            int counter,
+            boolean capableOfUserVerification,
+            ObjectConverter objectConverter) {
+        super(aaguid, attestationKeyPair, null, null, counter, capableOfUserVerification, objectConverter);
```
Comment (ynojima) [inline webauthn4j-test/src/main/java/com/webauthn4j/test/authenticator/webauthn/NoneAttestationAuthenticator.java:44]: As None attestation doesn't have signature, I think `attestationKeyPair` should be always `null`, and constructor parameter `attestationKeyPair` can also be removed.
```suggestion
        super(aaguid, null, null, null, counter, capableOfUserVerification, objectConverter);
```

Comment (majk-p) [reply webauthn4j-test/src/main/java/com/webauthn4j/test/authenticator/webauthn/NoneAttestationAuthenticator.java]: You're right! `attestationKeyPair` removed

**Review by ynojima — APPROVED** [review]

**Comment by ynojima** [comment]
Thank you for your pull-request! Now merged into master.

**Pull request merged**